ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -637,6 +637,12 @@ $defs:
             documentation_options:
               table: node-type-l2-mlag-configuration
             type: str
+          mlag_compact_addressing:
```

Comment:
Please remove this again, since we use the fabric_ip_addressing construct instead